stable/README.md

```diff
@@ -105,41 +93,49 @@ Client: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7
 Server: &version.Version{SemVer:"v2.14.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
 ```
 
-### Grant the Tiller Server Edit Access to the Current Project
-
-The Tiller server will probably need at least `edit` access to each project where it will manage applications. In the case that Tiller will be handling Charts containing Role objects, `admin` access will be needed.
-
+You can also verify that the Tiller server is running via `kubectl`.
 ```bash
-oc policy add-role-to-user edit "system:serviceaccount:${TILLER_NAMESPACE}:tiller"
+kubectl get pods -n kube-system
+NAME                                READY   STATUS    RESTARTS   AGE
+...
+tiller-deploy-8c5679674-k9c7m       1/1     Running   0          47m
+...
 ```
 
-The [blog article written by Red Hat][0] suggests having `Tiller` in its own project, separate from application projects. However, this is **NOT** how banks deploy Tiller, it is **NOT** best practice from a security perspective. Banks typically deploy Tiller in each application's project. That way:
 
-- the tiller `edit` role does not have to have `admin` access to the whole cluster
-- each project can be individually managed w.r.t. the version of `tiller` managing its deployments; e.g., no **"big bang"** during an upgrade.
+## Deploying NuoDB using Helm Charts
 
-So instead lets only run our applications alongside `tiller`, in the same project. To deploy a canary application, such as Node.js:
+### Grant OpenShift privileges
 
+First, you should create a new namespace for NuoDB.
 ```bash
-$ helm install https://github.com/jim-minter/nodejs-ex/raw/helm/helm/nodejs-0.1.tgz -n nodejs-ex
+oc new-project nuodb
 ```
 
-Verify that `tiller` properly launched Node.js.
-
+Create a new service account.
 ```bash
-$ oc get pods
-NAME                      READY     STATUS    RESTARTS   AGE
-nodejs-example-1-build    1/1       Running   0          1m
-tiller-86c4495fcc-rhq97   1/1       Running   0          15m
+kubectl -n nuodb create serviceaccount nuodb
 ```
 
-Now that we've seen Helm deploy an application successfully, we're confident in its installation and configuration, lets now move on to deploying NuoDB. But first, lets clean up Node.js:
+Next, you will want to give your new service account the correct SecurityContextConstraints to run NuoDB.
+You can find the recommended SecurityContextConstraints in `deploy/nuodb-scc.yaml`
 
 ```bash
-$ helm del --purge nodejs-ex
+oc apply -f deploy/nuodb-scc.yaml -n nuodb
+oc adm policy add-scc-to-user nuodb-scc system:serviceaccount:nuodb:nuodb -n nuodb
```

Comment:
My understanding is that we don't use `nuodb` service account currently.

Reply:
We do. See above. `kubectl -n nuodb create serviceaccount nuodb`

Reply:
You are creating it, but what I think Stanimir meant is that we don't then use it (at the moment).  To use it, the charts would need to supply serviceAccountName.  For the moment we are relying on the SCC being applied to default...